### ISSUE TITLE
[Vulnerability] Is AdGuard Home prone to log4shell?

### ISSUE THREAD

**Issue by privacy-advo** [AdguardTeam/AdGuardHome#3938]
Hi, 
are some AdGuard products, especially AdGuard Home, prone to CVE-2021-44228 aka log4shell?
Thanks

**Comment by sachinwadhwa** [comment]
It's "Log4j" which is Java based utility. Adguard is based on Go language and I don't think log4j is used.

**Comment by privacy-advo** [comment]
Yes, for sure. But it's always nice to have a feedback from the developers concerning their wider array of products.
